xscen/__init__.py

```diff
@@ -2,28 +2,28 @@
 import warnings
 
 # Import the submodules
-from . import (
-    aggregate,
-    biasadjust,
-    catalog,
-    checkups,
-    common,
-    config,
-    extraction,
-    finalize,
-    indicators,
-    io,
-    regridding,
-    scr_utils,
-)
+from . import catalog, checkups, config, extract, io, utils
```

Comment:
I would also import the other files, even if all functions are top-level?
```suggestion
from . import biasadjust, catalog, checkups, config, ensembles, extract, indicators, io, regrid, scripting, utils
```